code/game/machinery/lightswitch.dm

```diff
@@ -119,11 +119,6 @@
 	return ..()
 
 /obj/machinery/light_switch/attack_hand(mob/user)
-	toggle_switch()
-
-/obj/machinery/light_switch/proc/toggle_switch(var/newstate = null)
```

Comment:
You shouldn't be removing this

Reply:
It looks like a bugfix that was included in the previous PR, I'll amend it.

Reply:
I see, there was an issue with the revert function I was using. It weirdly tried to comment around it rather than remove the line of code, so I missed it while I was editing things myself. Thank you! As for why remove it I'm just trying to be atomic about things and remove the code entirely rather than piecemeal. It'll still exist in the history.

As for the fingerprinting, I don't know? I would have been able to check it when I was an admin, but I get the feeling it does leave fingerprints still.